src/js/components/Form/stories/Uncontrolled.js

```diff
@@ -58,6 +59,9 @@ const Example = () => {
             <FormField label="Age" name="age" pad>
               <RangeInput name="age" min={15} max={75} />
             </FormField>
+            <FormField label="Image" name="image">
+              <FileInput name="image" />
+            </FormField>
```

Comment:
The FileInput value isn't part of the Update result as shown in the console log (via drag and drop) seems to work ok with the `browse` option)

![image](https://user-images.githubusercontent.com/6320236/95129403-31bcd300-0718-11eb-9e23-d788d75a6a26.png)

Reply:
Should be fixed.